src/nvim/os/env.c

```diff
@@ -464,12 +464,14 @@ char_u *vim_getenv(char_u *name, bool *mustfree)
 
       /* remove "doc/" from 'helpfile', if present */
       if (p == p_hf)
-        pend = remove_tail(p, pend, (char_u *)"doc");
+        pend = (char_u *) remove_tail((char *)p, (char *)pend, (char *)"doc");
```

Comment:
Sorry about that - I was running into a weird issue and did this as part of troubleshooting and forgot to roll it back.  Fixed.